src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -733,14 +733,44 @@ impl<T: AstInfo> AstDisplay for CreateIndexStatement<T> {
             f.write_str(")");
         }
         if !self.with_options.is_empty() {
-            f.write_str(" WITH (");
+            f.write_str(" WITH ");
```

Comment:
why no more `(` / `)`? is that just the direction we're moving in?  This seems separate than the `()` -> `[]` change I see in a couple places

Reply:
IIRC the parens were necessary because of some ambiguity in the grammar; maybe there was something else that could be comma-separated after the `WITH` options. I think the pattern just propagated everywhere for consistency's sake. However, this should no longer be a concern if we're parsing options using keywords.

Reply:
Ah, mentioned below, but I think we should keep them in `CREATE INDEX` specifically to match PostgreSQL. Everywhere else we have keyword arguments, I think we should ditch the parens _and_  the `WITH`!